label_studio/data_export/serializers.py

```diff
@@ -30,6 +33,13 @@ class Meta:
           'completed_by': (CompletedBySerializer, )
         }
 
+    def get_result(self, obj):
+        # run frames extraction on param, result and result type
+        if obj.result and self.context.get('interpolate_key_frames', False) and \
+                is_video_object_tracking(obj.task.project.label_config):
```

Comment:
I don't think it's a good idea to call `parse_config` every time inside `is_video_object_tracking` - let's pass `parsed_label_config` here instead, or check for `<VideoRectangle>` substring in `label_config`